kanban-lite/src/aoe_kanban_lite/main.py

```
@@ -0,0 +1,246 @@
+            print("[kanban-lite] worker exiting", file=sys.stderr, flush=True)
+
+    def _stdin_reader(self) -> None:
+        """Read JSON-RPC messages from stdin and enqueue them."""
+        try:
+            while True:
+                msg = read_message(self.in_stream)
+                if msg is None:
+                    print("[kanban-lite] stdin EOF", file=sys.stderr, flush=True)
+                    break
+                self.inbound_queue.put(msg)
+        except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)